vars/cloudFoundryDeploy.groovy

```diff
@@ -196,11 +196,20 @@ private String deleteOptionIfRequired(Map config) {
     }
 }
 
-private String stopOldAppIfRequired(Map config) {
+private void stopOldAppIfRequired(Map config) {
```

Comment:
The name suggests that it checks if it is required to stop the app or what is the requirement?

Maybe stopOldAppIfRunning ...still suggests that it checks if it is running but clearer that just required.

Reply:
done